specification/specs/cat/cat_aliases/CatAliasesRequest.ts

```diff
@@ -4,7 +4,7 @@ class CatAliasesRequest extends CatRequestBase {
     name?: Names;
   }
   query_parameters?: {
-    expand_wildcards?: ExpandWildcards;
+    expand_wildcards?: ExpandWildcards | Array<ExpandWildcards>;
```

Comment:
I think the better fix would be:

```
export enum ExpandWildcardsOptions {
..
}

export type ExpandWildcards = ExpandWildcardsOptions | Array<ExpandWildcardsOptions> 
```

This should fix all usages of `ExpandWildcards`